ere-guests/Cargo.toml

```diff
@@ -181,11 +181,11 @@ k256 = { version = "0.13", default-features = false, features = [
     "serde",
     "pem",
 ] }
-sparsestate = { git = "https://github.com/eth-act/zkvm-ethereum-mpt.git", tag = "v0.1.0" }
+sparsestate = { git = "https://github.com/eth-act/zkvm-ethereum-mpt.git", rev = "96070450f545277a6238c6409d1294d9bca66c16" }
```

Comment:
I needed to [update the extracted-MPT repo](https://github.com/eth-act/zkvm-ethereum-mpt/pull/5) to the same Reth `main` github commit that we based our Reth-fork, because of dependency issues. 
I tried to use the latest Reth tag, but wasn't enough so had to push it to the right (newer) `master` commit.

I don't think this is a big deal -- mainly a consequence that we're basing our Reth-fork on stuff newer than the lastest release.